README.md

```diff
@@ -1,4 +1,7 @@
-# About gunk
+[![GoDoc][card-godoc]][godoc]
+[![Build Status][card-travis]][travis]
```

Comment:
Wow, what kind of black magic is this? I always thought one needed to include the full link for these to work.

Reply:
Reading the end of the markdown, it now dawns on me. I don't think it makes sense to separate these links. These are single-line badges - I personally think that `[links]` only make sense within paragraphs, to keep them more readable.